lib/src/tasks/local/config.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+library dart_dev.src.tasks.local_commands.config;
+
+import 'package:dart_dev/src/tasks/config.dart';
+
+class LocalConfig extends TaskConfig {
+  List<String> taskPaths    = ['tool'];
+  String commandFilePattern = '([a-zA-Z0-9]+)_task.([a-zA-Z0-9]+)';
```

Comment:
Take it or leave it, but it might be nice to provide a `suffix` attribute or something to allow `_task` to be changed without rewriting the regex.